src/NServiceBus.Core.Tests/GlobalTestSetup.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NServiceBus.Core.Tests
+{
+    using System.Globalization;
+    using System.Threading;
+    using NUnit.Framework;
+
+    [SetUpFixture]
+    public class GlobalTestSetup
```

Comment:
Out of curiosity: Why didn't we centralize those settings in one file together with https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core.Tests/RedirectHelper.cs ? @Particular/nservicebus-maintainers